app/helpers/reactjs_helper.rb

```diff
@@ -24,7 +21,25 @@ def webpacked_plugins_with_global_js
   end
 
   def webpacked_plugins_css_for(*plugin_names)
-    css_tags_for(select_requested_plugins(plugin_names)).join.html_safe
+    Foreman::Deprecation.deprecation_warning('3.12', '`webpacked_plugins_css_for` is deprecated, plugin css is already loaded.')
```

Comment:
small todo: return nil after this line as sometimes the deprecation warning is shown in the dom above the foreman header